.codespellignore

```diff
@@ -0,0 +1,4 @@
+fo
+raison
```

Comment:
This comes from:
> .//interpreter/php/decode_amd64.c:72:  // The raison d'etre for this function is described in the php8 unwinding doc,

While this is understandable, should we instead use "professionnal" english?

Reply:
I thought about exactly this, but then it seemed that `raison d'etre` is a well-known term in English. But now that we are two people already who think about rewording, I'll do it :)

Reply:
done